services/director-v2/src/simcore_service_director_v2/models/schemas/constants.py

```diff
@@ -12,7 +12,7 @@
 # - itisfoundation
 # - 10.0.0.0:8473 (IP & Port)
 DYNAMIC_SIDECAR_DOCKER_IMAGE_RE = (
-    r"(^([a-zA-Z0-9:.-]+)/)?(dynamic-sidecar):([a-zA-Z0-9.-]+$)"
+    r"(^([_a-zA-Z0-9:.-]+)/)?(dynamic-sidecar):([_a-zA-Z0-9.-]+$)"
```

Comment:
Please run the test i attached with and without this chanve and add more string samples